libbeat/filter/condition.go

```diff
@@ -11,247 +11,308 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
+type RangeValue struct {
+	gte *float64
+	gt  *float64
+	lte *float64
+	lt  *float64
+}
+
+type EqualsValue struct {
+	Int uint64
+	Str string
+}
+
 type Condition struct {
```

Comment:
having multiple conditions how are they 'combined'? 'Condition' acts like an AND ?